lm_eval/tasks/unitxt/template.yaml.file

```diff
@@ -0,0 +1,20 @@
+group: 
+  - unitxt
+task: TASK_PLACEHOLDER
+dataset_path: unitxt/data
+dataset_name: card=CARD_PLACEHOLDER,template=TEMPLATE_PLACEHOLDER
+output_type:  generate_until
+training_split: train
+validation_split: test
+doc_to_text: "{{source}}"
+doc_to_target: target
+process_results: !function unitxt_wrapper.process_results
+generation_kwargs:
+  until:
+    - "</s>"
+metric_list:
+  - metric: METRIC_PLACEHOLDER
+    aggregation: unitxt
+    higher_is_better: true
+metadata:
+  version: 1.0
```

Comment:
Is this template used by the tasks here?

Reply:
This is used to generate the  tasks.classification.multi_class.yaml  and other yamls by the generate_yamls script.   I think I can also embed it in the script to avoid the extra file.